app/views/katello/api/v2/activation_keys/product_content.json.rabl

```diff
@@ -8,8 +8,7 @@ child @collection[:results] => :results do
   end
 
   child :content => :content do
-    attributes :id, :name, :label
-    attribute :contentUrl => :content_url
+    attributes :id, :name, :label, :content_url
```

Comment:
I would leave 'contentUrl' for api compatibility

Reply:
but you can add the new attribute as well

Reply:
Will add the new attributes, but the change I made here *was* to keep the API consistent. The original `attribute :contentUrl => content_url` renamed contentUrl to content_url in the rendered JSON

Reply:
oh right! carry on!